composer.json

```diff
@@ -12,9 +12,7 @@
     }
   ],
   "require": {
-    "php": ">=5.6"
-  },
-  "require": {
+    "php": ">=7.0",
     "phpunit/phpunit": "5.*"
```

Comment:
@gabriel403 @ziadoz Should this be `require-dev` **instead of** `require`, for both php and phpunit?